tests/snappi_tests/multidut/pfcwd/files/pfcwd_multidut_basic_helper.py

```diff
@@ -63,9 +63,17 @@ def run_pfcwd_basic_test(api,
     pytest_assert(testbed_config is not None, 'Fail to get L2/3 testbed config')
 
     start_pfcwd(duthost1, rx_port['asic_value'])
-    enable_packet_aging(duthost1)
     start_pfcwd(duthost2, tx_port['asic_value'])
-    enable_packet_aging(duthost2)
+    if ("platform_asic" in duthost1.facts and duthost1.facts["platform_asic"] == "broadcom-dnx"):
```

Comment:
Hi Shawn, 

Added code to check if the multi_asic is present and if true, enable PFCWD and packet-aging based on ASIC. If not, it will enable the same without ASIC-check.

Thanks,
-A